### DISCUSSION TITLE
Connect table is not working even after Algolia integration

### DISCUSSION THREAD

**Discussion by hariprasadms** [AntlerVC/firetable#463] [Support / Q&A] [answered]
I have installed the Algolia in firestore I can see that now collection appears in the Algolia but when I try to use connect table filed type it does not populate the data from collection.

Is there any configuration should be done in Firetable side to enable Algolia ?

**Comment by harinij** [comment, marked as answer]
We have an improved connect table option now that can be used without Algolia. This is available via "Connector" field type. The documentation for this is available [here](https://docs.rowy.io/field-types/connector)
